packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts

```diff
@@ -2,13 +2,13 @@ import { BindingsFactory } from '@comunica/bindings-factory';
 import { ActorQueryOperation } from '@comunica/bus-query-operation';
 import { KeysInitQuery } from '@comunica/context-entries';
 import { ActionContext, Bus } from '@comunica/core';
+import * as sparqlee from '@comunica/sparqlee/lib';
+import { isExpressionError } from '@comunica/sparqlee/lib';
```

Comment:
@rubensworks is this additional `/lib` wanted?

Reply:
No :-)